### DISCUSSION TITLE
Windows Package Manager 1.7.10861

### DISCUSSION THREAD

**Discussion by ryfu-msft** [microsoft/winget-cli#4326] [Releases]
This release is the third stable release of Windows Package Manager 1.7 for Windows 10 (1809+), and Windows 11.
Experimental features have been disabled in this release.

This release fixes issues related to the `EnableWindowsPackageManagerCommandLineInterfaces` Policy blocking COM calls which should be allowed when the CLI interface is disabled.

## What's Changed
* Fix EnableWindowsPackageManagerCommandLineInterfaces Policy Behavior by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4293
* Ignore corrupted source data by @johnmcpms in https://github.com/microsoft/winget-cli/pull/4291
* Better server start coordination by @johnmcpms in https://github.com/microsoft/winget-cli/pull/4292

**Full Changelog**: https://github.com/microsoft/winget-cli/compare/v1.7.10661...v1.7.10861

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.7.10861'>Windows Package Manager 1.7.10861</a>.</em>

**Comment by JuhaszBalint2** [comment]
Hello. I made a comment a couple of hours ago but I've deleted it since I have since fixed it and my scripts runs winget as SYSTEM with winget doing what it is supposed to do no problem, so sorry about that.

I'm also sorry if this is the wrong place to enquire about this, but are there software that run the winget client as SYSTEM (NT AUTHORITY\SYSTEM) by default? I'm just wondering if I'll always have to script this or if there's 'simpler' ways around that, too.

**Reply by Masamune3210** [reply]
Hello, spam account/bot with a stolen account token
